app/assets/styles/styles.css

```diff
@@ -35,156 +71,270 @@ body {
 
 .home-hero h1 {
 	/* Text styles*/
-	font-size: 5em;
+	font-size: 2.3em;
 
 	/* Reducing space between title and subtitle */
 	margin-top: 0;
 	margin-bottom: 0.5em;
+
+}
+
+.home-hero h2 {
+	/* Text styles*/
+	font-size: 0.8em;
 }
 
 /*Styling the button. we need to give the button a display type of block (<a> elements are inline by default) so that it will behave like other elements. We also need to give it a width - otherwise, it will take up the full width of the screen */
 .home-hero .btn {
 	/*positioning and sizing */
 	display: block;
-	width: 250px;
+	width: 120px;
 
 	/*Padding and margins. margin-left and margin-right center the btn. */
-	padding: 1em;
-	margin-top: 50px;
+	padding: 0.4em;
 	margin-left: auto;
 	margin-right: auto;
 
 	/* Border styles*/
 	border: 3px solid white;
-	border-radius: 20px;
+	border-radius: 4px;
 
 	/*Text styles */
 	color: white;
-	font-size: 1.5em;
-
-}
-img {
-	max-width: 100%;
-	height: auto;
-}
-
-a {
-	text-decoration: none;
-}
+	font-size: 0.85em;
 
-p {
-	line-height: 1.55;
 }
 
 /*Positioning and sizing the main content, setting it to be inside 75% of the browser width. */
 .post-wrapper {
-  overflow: hidden;
-  padding-left: 18px;
-  padding-right: 18px;
-  max-width: 75vw;
-  margin: 80px auto;
-  font-size: 1.15rem;
+	overflow: hidden;
+	padding-left: 18px;
```

Comment:
Lots of mixed units. I would recommend using 

- rem for fonts
- em for padding/margins
- pixels for widths or precise units